pom.xml

```diff
@@ -714,14 +714,19 @@
       <dependency>
         <groupId>org.json4s</groupId>
         <artifactId>json4s-jackson_${scala.binary.version}</artifactId>
-        <version>3.5.3</version>
+        <version>3.6.5</version>
```

Comment:
What does this improve?

Reply:
1. It uses the same Scala version as spark.
2. It fixes issues with default value used in case class constructor.  
3. and some other improvements.